www/src/theme.js

```diff
@@ -237,7 +238,7 @@ const sharedStyles = {
     '& p > code, & li > code': {
       background: hex2rgba(colors.note, 0.3),
       padding: '0 3px',
-      fontSize: 16,
+      fontSize: 'inherit',
```

Comment:
IMO, the font size of `<code>` should not be hardcoded as 16 because it is used in multiple places with different font size:

### Prev

<img width="400" alt="screen shot 2017-10-05 at 12 05 15 am" src="https://user-images.githubusercontent.com/1315101/31188486-a73fbfa0-a967-11e7-9d63-3ad00c69d9ec.png">

### Curr

<img width="400" alt="screen shot 2017-10-05 at 12 06 28 am" src="https://user-images.githubusercontent.com/1315101/31188487-a7483dec-a967-11e7-8765-93d014fbd63d.png">